mock-api/msw/handlers.ts

```diff
@@ -246,6 +246,20 @@ export const handlers = makeHandlers({
     return paginated(query, ips)
   },
   floatingIpView: ({ path, query }) => lookup.floatingIp({ ...path, ...query }),
+  floatingIpUpdate: ({ path, query, body }) => {
+    const floatingIp = lookup.floatingIp({ ...path, ...query })
+    if (body.name) {
+      // only check for existing name if it's being changed
+      if (body.name !== floatingIp.name) {
+        errIfExists(db.floatingIps, { name: body.name })
```

Comment:
needs `project_id: floatingIp.project_id` to enforce name uniqueness for the project only